clients/lcdproc/disk.c

```diff
@@ -59,7 +59,7 @@ disk_screen(int rep, int display, int *flags_ptr)
 
 	/* Holds info to display (avoid recalculating it) */
 	struct disp {
```

Comment:
This function seems broken in many ways. Most notably table is not static, so it is recalculated anyway.

Maybe just merge the two for loops together and get rid of table completely?